src/tools/rust-analyzer/crates/hir-def/src/attr/builtin.rs

```diff
@@ -262,24 +262,6 @@ pub const INERT_ATTRIBUTES: &[BuiltinAttribute] = &[
         naked_functions, experimental!(naked)
     ),
 
-    // Plugins:
```

Comment:
Given that this is a no-op and we're currently [unable](https://github.com/rust-lang/rust/pull/116136#issuecomment-1733307521) to do syncs, would you mind dropping this change and optionally filing it against the RA repo?

Reply:
Sure, if this PR is approved I can do that.